build.proj

```diff
@@ -33,8 +33,8 @@
       RestoreProjects;
       BuildRedist;
       BuildNative;
-      $(TraversalBuildDependsOn);
```

Comment:
Is this change necessary?

Reply:
If we dont do this, then we have to build the Microsoft.ML.Benchmark project again before running

Reply:
dotnet run will build the project again so its safe to revert it